client/rust/src/contract/mod.rs

```diff
@@ -85,8 +85,8 @@ impl ContractData {
     }
 }
 
-#[rsharp::substitute("tp_client::contract")]
 #[cfg(feature = "c_api")]
+#[rsharp::substitute("tp_client::contract")]
```

Comment:
Why was this moved? Is it style or does it affect the functionality?

Reply:
style, I preferred the cfg gate at the top. Technically attribute order can also affect functionality, I believe the outermost attribute executes first. But for a cfg macro it disables not just the item its on but all the attribute macros anyway, so it doesn't matter here.